src/lifecycle.py

```diff
@@ -0,0 +1,31 @@
+from fabric.api import task
+from buildercore import lifecycle
+from decorators import requires_aws_stack, timeit, echo_output
+
+@task
+@requires_aws_stack
+@timeit
+def start(stackname):
+    lifecycle.start(stackname)
+
+@task
+@requires_aws_stack
+@timeit
+def stop(stackname):
+    lifecycle.stop(stackname)
+
+@task
+@requires_aws_stack
+@echo_output
+def last_start_time(stackname):
+    return lifecycle.last_start_time(stackname)
+
+@task
+@requires_aws_stack
+@timeit
+def stop_if_next_hour_is_imminent(stackname, minimum_minutes='55'):
+    # TODO: can we write a description of the @task somewhere?
```

Comment:
Like this?
http://docs.fabfile.org/en/1.12/api/core/tasks.html#fabric.tasks.Task